layouts/partials/head.html

```diff
@@ -55,6 +55,13 @@
 {{- $eurekaCSS := resources.Get "css/eureka.css" | postCSS (dict "config" "./assets/css/postcss.config.js") }}
 {{- $eurekaCSS := $eurekaCSS | minify }}
 <link rel="stylesheet" href="{{ $eurekaCSS.Permalink }}">
+{{ if isset $.Params "carousel" }}
+{{ if $.Params.carousel.enable }}
+<link preload rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.9.3/tiny-slider.css">
```

Comment:
The if conditions can be simplified as described above. And please define css in `data/assets.toml` and call it here.

Reply:
Conditions simplified in fabf93a 

Defined in `assets.toml` in c1e6f71